clang/test/dpct/cmake_migration/case_001/expected.txt

```diff
@@ -11,7 +11,7 @@ include(${dpct_cmake_file_path})
 set (CMAKE_CXX_STANDARD 17)
 set (CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
-find_package(IntelSYCL REQUIRED)
+dpct_helper_find_package(IntelSYCL REQUIRED)
```

Comment:
After a further thought, find_package() is not a cuda specific cmake syntax, so to introduce dpct_helper_find_package is a little over designed.
we can map cuda specific argument used in find_package() to sycl mappings if the mapping is available, or "" if mapping is not available.

As to mapping argument related to MKL, we can introduce a help variable like [DNN_LIB ](https://github.com/oneapi-src/SYCLomatic/blob/SYCLomatic/clang/tools/dpct/cmake/dpct.cmake#L95C9-L95C15) to simple migration implementation.